src/api/config/routes.rb

```diff
@@ -329,7 +330,6 @@ def self.public_or_about_path?(request)
       get 'user/register_user' => :register_user
 
       get 'user/login' => :login
-      post 'user/logout' => :logout
```

Comment:
you say in your commit message

> but keeping the former ones for backward compatibility.

but you are removing them, right? :confused:

Reply:
I only restored the `get` routes, as was requested in https://github.com/openSUSE/open-build-service/pull/4731#discussion_r177054886.

Shall I also restore `post` route or reword the commit?

Reply:
I think we need to keep all the routes, because other services can be using them. So, restore `post` :wink: